File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/navdrawer/NavDrawerActivity.java
package com.jdroid.android.sample.ui.navdrawer;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class NavDrawerActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return NavDrawerFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/navdrawer/NavDrawerActivity.kt
package com.jdroid.android.sample.ui.navdrawer

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class NavDrawerActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return NavDrawerFragment::class.java
    }
}
